### DISCUSSION TITLE
My `quarto-letter` extension now supports typst

### DISCUSSION THREAD

**Discussion by produnis** [quarto-dev/quarto-cli#11392] [Show and Tell]
### Description

Hi everyone, I just wanted to show off a bit: my `quarto-letter` extension now also supports Typst!

Here's a screenshot:

![](https://i.imgur.com/ubANbwb.png)

The letter format is optimized for German standards, specifically DIN-A4, but all parameters can be adjusted via YAML.

This is my first "official" Typst extension, and as a self-taught hobbyist programmer, I’ve probably coded some things in a stupid or unconventional way. But, everything works as intended!
Find my repo here: <https://github.com/produnis/quarto-letter>

Typst is really amazing- its rendering speed is outstanding (So, can't wait to see typst support for `quarto-book` ;-) ).

Many thanks to all the developers!

**Comment by mcanouil** [comment]
Thanks for sharing!

You might want to submit your extension to https://quarto.org/docs/extensions/ (additionally to https://m.canouil.dev/quarto-extensions/, although submitting your extension to the first will propagate on it).
For better discovery on GitHub and https://m.canouil.dev/quarto-extensions/, I suggest you update your repository description and to add topics to your repository.

**Reply by produnis** [reply]
I added a CC4 license to my repo, because I am used to it for my teaching material. Is this a "proper" license for extensions as well?

**Reply by mcanouil** [reply]
That's really up to you and what you want/allow others to do with your extension.
The top licenses used for extension so far:
1. MIT license
2. CC licenses
3. GPL

(technically no licensing at all is number 2)

**Reply by produnis** [reply]
thx, I changed mine to MIT as well.. :)
